src/uniparc/components/landing-page/LandingPage.tsx

```diff
@@ -0,0 +1,292 @@
+          )}
+        >
+          <h2>Databases</h2>
+          <div className={styles['table-container']}>
+            {groupedDatabases.map((databases, index) => (
+              // eslint-disable-next-line react/no-array-index-key
+              <table key={index}>
+                <thead>
+                  <tr>
+                    <th>Cross-reference</th>
+                    <th>
+                      Number of
+                      <br />
+                      UniParc entries
+                    </th>
+                  </tr>
+                </thead>
+                <tbody>
+                  {databases.map(({ count, value, label }) => (
+                    <tr key={value}>
+                      <td>
+                        <Link
+                          to={{
+                            pathname: LocationToPath[Location.UniParcResults],
+                            search: stringifyQuery({
+                              query: '*',
+                              facets: `${FacetsEnum.Database}:${value}`,
+                            }),
+                          }}
+                        >
+                          {label || value}
+                        </Link>
+                      </td>
+                      <td className={styles.end}>
+                        <LongNumber>{count}</LongNumber>
+                      </td>
+                    </tr>
+                  ))}
+                </tbody>
+              </table>
+            ))}
+          </div>
+
+          <Button
+            variant="tertiary"
+            onClick={() => setExpanded((expanded) => !expanded)}
+          >
+            {expanded ? 'View less' : 'View more'} databases
+          </Button>
+        </section>
+
+        {/* Text */}
+        <section
+          className={cn(
+            'uniprot-grid-cell--small-span-12',
+            `uniprot-grid-cell--medium-span-${expanded ? '9' : '5'}`
+          )}
+        >
+          <h2>Searching the Database</h2>
+          <br />
+          <p>
+            UniParc provides text- and sequence-based searches. Performing a
+            sequence similarity search against UniParc is equivalent to
+            performing the same search against all databases cross-referenced in
+            UniParc, as UniParc contains all proteins from its source databases.
+          </p>
+          <p>
+            <Link
+              to={{
+                pathname: LocationToPath[Location.Blast],
+                search: stringifyQuery({
+                  database: 'uniparc',
+                }),
+              }}
+            >
+              Start a sequence similarity search in UniParc using BLAST »
+            </Link>
+          </p>
+        </section>
+
+        {/* Downloads */}
+        <section className="uniprot-grid-cell--small-span-12 uniprot-grid-cell--medium-span-3">
+          <h2>Download</h2>
+          <div className={styles.download}>
+            <br />
+            <p>
```

Comment:
Seeing a `<ul> cannot appear as a descendant of <p>.` error so I guess just changing this to `<div>` would solve it but there is a little spacing difference seen in the second screenshot.
<img width="2032" alt="Screenshot 2024-07-23 at 15 14 16" src="https://github.com/user-attachments/assets/3b7c259c-a87b-4a5d-bf6c-b633c5c00384">
<img width="2032" alt="Screenshot 2024-07-23 at 15 14 07" src="https://github.com/user-attachments/assets/983ee695-bd45-498c-ab50-0e1efa6659bf">

Reply:
In the end I just replaced with the div, it took me a bit to see the difference so I'm assuming a user wouldn't be too bothered